modules/mw.cx.TranslationMdwiki.js

```
@@ -199,124 +227,202 @@ async function get_from_medwiki_or_mdwiki_api(title, tr_type) {
-		let categories = add_sw_categories(result.segmentedContent);
-		result.categories = categories;
+	mdwiki_last_url.url = "";
+	// ---
+	// if page_title start with "Video:" then tr_type = all
+	if (page_title.startsWith("Video:") || page_title.startsWith("video:")) {
+		tr_type = "all";
+	}
+	// ---
+	const result = await fetchSourcePageContent_mdwiki_new(page_title, targetLanguage, tr_type, user_name);
+	// ---
+	if (result && result.segmentedContent && targetLanguage === "sw") {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)